src/test/java/com/puppycrawl/tools/checkstyle/checks/javadoc/JavadocPackageCheckTest.java

```diff
@@ -123,4 +131,21 @@ public void testAnnotation() throws Exception {
             getPath("annotation"
                     + File.separator + "package-info.java"), expected);
     }
+
+    // For 100% coverage
+    @Test
+    public void testThrowsCorrectExceptionIfFailedToGetCanonicalFile() throws Exception {
+        final JavadocPackageCheck check = new JavadocPackageCheck();
+        final File mockFile = mock(File.class);
+        doThrow(IOException.class).when(mockFile).getCanonicalFile();
```

Comment:
Please don't just throw an exception when `getCanonicalFile()` is called, but reproduce a situation(s) when this exception can be thrown.

Reply:
Changed implementation so there is no need in this test now. Deleted it.